spec/fizzbuzz_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'solver'
+
+RSpec.describe Solver do
+  before :each do
+    @solver = Solver.new
+  end
+
+  describe 'fizzbuz 3' do
+    it 'return fizz' do
+      result = @solver.fizzbuzz(3)
+      expect(result).to eql('fizz')
+    end
+  end
```

Comment:
* Kindly fix this errors I get when I run your tests 
![image](https://github.com/ichsansandy/tdd-ruby/assets/52497246/a219a66c-c38d-45b9-8b85-71976e1c0fd3)